db/seeds.d/50-bootdisk_templates.rb

```diff
@@ -1,33 +1,33 @@
-kind = TemplateKind.find_or_create_by_name('Bootdisk')
+kind = TemplateKind.find_or_create_by(:name => 'Bootdisk')
```

Comment:
.find_or_create_by isn't a valid method on Rails 3, so you'll either be limiting the plugin to Rails 4 or you need to switch to a compatible syntax such as .where(..).first_or_create.